tests/iostability/test_iostability_degraded_path.py

```diff
@@ -94,11 +94,10 @@ def test_object_crud_single_pod_failure(self):
 
         self.log.info("Step 2: Shutdown the data pod safely by making replicas=0,"
                       "check degraded status.")
-        resp = self.ha_obj.delete_single_pod_setting_replica_0(self.master_node_list[0],
-                                                               self.hlth_master_list[0],
-                                                               POD_NAME_PREFIX)
-        assert_utils.assert_true(resp[0], resp[1])
-        self.log.info("Deleted pod : %s",resp[1])
+        resp = self.ha_obj.delete_kpod_with_shutdown_methods(self.master_node_list[0],
+                                                            self.hlth_master_list[0])
+        assert_utils.assert_true(resp[0], "Failed in shutdown or expected cluster check")
```

Comment:
Please modify assert as per latest changes.

Reply:
> Please modify assert as per latest changes.

here, they are not re-storing the pod. Hence this test does not need to get re-store info. I have already checked with @priyankaborawake, she said any kind of failure will be treated as failure. Since its long running test, test don't want it to recover.